### DISCUSSION TITLE
Specifying path of preset yaml file

### DISCUSSION THREAD

**Discussion by racng** [milaboratory/mixcr#1046] [Q&A] [answered]
Is there a way to specify a local custom preset without requiring it to be in the same working directory?
For example, if I have the `custom_preset.yaml` file stored at a folder with path `/home/user/folder/`, the command
`mixcr analyze -preset local:/home/user/folder/custom_preset ...` doesn't work. I would have to copy the yaml to the working directory and run `mixcr analyze -preset local:custom_preset`.

If there is no way to do this, I would be totally fine with copying the `yaml` file. I am not requesting a new feature here. 

Thank you!

**Comment by Alex-Davydov** [comment, marked as answer]
Hi,
You can put your preset in `/home/user/.mixcr/presets/ ` and run `mixcr analyze --preset local:custom_preset ...`
Alex
